### ISSUE TITLE
Change nMaxTipAge for testnet to make it easier for users to create a private testnet

### ISSUE THREAD

**Issue by bitcartel** [zcash/zcash#2273]
Resolve user issue https://github.com/zcash/zcash/issues/1802 by setting nMaxTipAge to 0x7fffffff

**Comment by daira** [comment]
This can be done by reverting https://github.com/zcash/zcash/commit/5551db88224c9c9e70b8bcf5f25baaaaad656eac .

**Comment by daira** [comment]
We definitely want to make it easier to create private testnets. However, I think the problem here was the interaction of three things:
a) low hash rate (or a delay in initially setting up mining on a new testnet);
b) the nMaxTipAge rule;
c) the fact that we had removed the min-difficulty rule for all testnets.

c) was fixed in #3559. a) could be addressed by automation to set up a testnet in such a way that mining starts immediately. So it might not be necessary to change b).
